static/styles/pygments.scss

```diff
@@ -213,7 +213,7 @@
 
 
 /* Syntx Highlighting for night-mode */
-body.night-mode {
+body.night-mode .message_content {
```

Comment:
Check whether this breaks the "message formatting" documentation -- we might need to add a `message_content` selector there.